hindi/historical-places/taj-mahal.md

```diff
@@ -0,0 +1,3 @@
+ताजमहल (उर्दू: تاج محل) भारत के आगरा शहर में स्थित एक विश्व धरोहर मक़बरा है। इसका निर्माण मुग़ल सम्राट शाहजहाँ ने, अपनी पत्नी मुमताज़ महल की याद में करवाया था।
+सन् १९८३ में, ताजमहल युनेस्को विश्व धरोहर स्थल बना। ताजमहल को भारत की इस्लामी कला का रत्न भी घोषित किया गया है।
+मुख्य पृष्ठ पर जाने के लिए [यहां](create-your-own-adventure/hindi/bollywood.md)क्लिक करें।
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.